tests/test_launch_methods/test_jsrun/test_5/cmds

```diff
@@ -0,0 +1,3 @@
+# Task needs to be assigned nodes 1 and 2, physical core 1 of each node, HW thread 1 of each physical core
+export OMP_NUM_THREADS=1
+jsrun -a 1 -U resource-file /ccs/proj/csc190/Hello_jsrun/jsrun_layout
```

Comment:
You are missing `verbose`, otherwise the node name will never appear in the output

Reply:
I am not sure I follow. These were the exact commands that were run and I do get the node name/ID. Let me know if you think I missed something.